src/components/EventItem/EventItem.js

```diff
@@ -17,6 +18,18 @@ class EventItem extends React.Component {
     deleteSingleEvent(event.id);
   }
 
+  editEvent = (e) => {
+    e.preventDefault();
+    const { passEventToEdit, event } = this.props;
+    passEventToEdit(event.id);
+  }
+
+  listingClick = (e) => {
```

Comment:
What is this function doing?

Reply:
Actually.. It's not doing anything!  I think it eventually would be so you could click on the event and get the ID, right?  It's where I was trying to figure things out by using the Zillione project as an example.  So it's basically a dead function at the moment.